lti_consumer/lti_xblock.py

```diff
@@ -656,7 +656,11 @@ def lti_provider_key_secret(self):
         """
         for lti_passport in self.course.lti_passports:
             try:
-                lti_id, key, secret = [i.strip() for i in lti_passport.split(':')]
+                # NOTE While upacking the lti_passport by using ":" as delimiter, first item will be lti_id, last item will be client_secret and the rest are considered as client_key. So you can have more than one colon for client_key.
```

Comment:
The comment looks fair enough but too long, please split your comment into several lines limited by 120 characters. This and other recommendations are in the [OpenEdx Style Guide](https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#syntax-and-organization).
As a pycharm admirer, I would prefer to add one more space after `#` in the second and following comment's lines which make IDE highlight the comment as one block. At the same time, I doubt that we have such an approach as a common practice so it's up to you.

Reply:
Hi @idegtiarov 

Shall I use docstrings like this 
```
"""
While unpacking the lti_passport by using ":" as delimiter, first item will be lti_id, 
last item will be client_secret and the rest are considered as client_key.
So you can have more than one colon for client_key.
"""
```

Reply:
Please use inline comments if you want to have comments as much close as possible to the change in the code.
```
# NOTE While unpacking the lti_passport by using ":" as delimiter, first item will be lti_id, 
#  last item will be client_secret and the rest are considered as client_key.
#  So you can have more than one colon for client_key.
```

Multiple docstrings will work perfectly in case you'd like to add details in the function description.

```
"""
Obtains client_key and client_secret credentials from current course.

While unpacking the lti_passport by using ":" as a delimiter, the first item will be lti_id, 
the last item will be client_secret and the rest are considered as client_key.
So you can have more than one colon for client_key.
"""
```

Reply:
I'd like to place them near to the code. Sorry for the delay, also thanks for all the knowledge sharing. I really appreciate it. I have pushed the fixes, please take a look.